src/services/authService.js

```diff
@@ -52,9 +52,12 @@ class AuthService {
   }
 
   login = () => {
-    cfaSignIn("google.com").subscribe((user) => {
-      console.log(user);
-    });
+    cfaSignIn('google.com').subscribe(
+       (user) => {
+        console.log(user);
+        localStorage.setItem('loggedin','loggedintoken'); //For test loggedin is stored
```

Comment:
you are setting `loggedin` to true when user is logging in, but no changes in this value when the user logged out, So if user logged In and then he logged out, there will be `loggedIn` state still true, which is not ideal.